genesyscloud/group_roles/resource_genesyscloud_group_roles_schema.go

```diff
@@ -78,3 +81,27 @@ func GroupRolesExporter() *resourceExporter.ResourceExporter {
 		},
 	}
 }
+
```

Comment:
Probably should make this part of the proxy.  Thoughts

Reply:
I agree unless we plan on refactoring it to another package instead of duplicating - in that case I don't think it's necessary to bake it into the proxy